pom.xml

```diff
@@ -334,6 +335,13 @@
                     </execution>
                 </executions>
             </plugin>
+
+            <plugin>
+                <groupId>org.jacoco</groupId>
+                <artifactId>jacoco-maven-plugin</artifactId>
+                <version>${jacoco.version}</version>
+            </plugin>
```

Comment:
Do the plugin coordinates need to be declared here? Seem like doing it once in code-coverage profile should be enough.

Reply:
@miroslavpojer ?

Reply:
You are right. It can be removed from build/plugins section. Tested locally. Current presence is not source of failure. There is planned another related improvement - excludes. I will remove it here. Are you ok with that?

Reply:
yes, thanks